internal/bminventory/inventory.go

```diff
@@ -458,6 +458,13 @@ func (b *bareMetalInventory) RegisterClusterInternal(
 		monitoredOperators = append(monitoredOperators, newOLMOperators...)
 	}
 
+	platform := params.NewClusterParams.Platform
+	if models.PlatformTypeBaremetal == *platform.Type && *params.NewClusterParams.UserManagedNetworking {
+		*platform.Type = models.PlatformTypeNone
+	} else if models.PlatformTypeNone == *platform.Type {
+		params.NewClusterParams.UserManagedNetworking = swag.Bool(true)
```

Comment:
I think you want to check that that params.NewClusterParams.UserManagedNetworking is nil before overriding it